Source/DataSources/KmlDataSource.js

```diff
@@ -1140,6 +1146,8 @@ define([
 
         if (!defined(billboard.image)) {
             billboard.image = dataSource._pinBuilder.fromColor(Color.YELLOW, 64);
+        } else if (billboard.image == false) {
```

Comment:
@josh-bernstein this is why CI failed on eslint.  `==` should be `===`

Reply:
@hpinkos This condition is only true in the desired cases if I use ==, not ===.  
I could remove this 'else if' block altogether and it will still function correctly.  I just think it is more intuitive to set the value back from false to undefined

Reply:
Do `} else if (!billboard.image) {` instead

Reply:
Or what is the value of `billboard.image` here?  Is it `'false'` instead of `false`?  
In that case, do `} else if (billboard.image === 'false') {`

Reply:
You explicitly set `billboard.image` to `false`. `===` should work in this case because the type and value will match.

Reply:
@hpinkos @tfili I agree it seems like it should be false.  However, this is the value I get when using my browser's debugger.

![billboard](https://user-images.githubusercontent.com/16944793/31910898-f07034d0-b80c-11e7-9dc1-32c835ccaeba.PNG)

Reply:
Oooh okay.  It's already being converted to an `Property` at this point.
In this case, let's do `} else if (billboard.image instanceof ConstantProperty && !billboard.image.getValue()) {`.

Please also add a comment like you did in your change above so we remember why we did this.

Reply:
@hpinkos `billboard.image instanceof ConstantProperty` throws error saying ConstantProperty is undefined.  However, everything seems to work if I do `} else if (billboard.image && !billboard.image.getValue()) {'

Reply:
At the top of the file, require in `ConstantProperty`.  See how we require in `SampledPositionProperty` in this file for an example.  We want to make sure it's a constant property because other property types require an argument passed into `getValue`

Reply:
@hpinkos In what other situation would billboard.image be defined but have a value of false?

Reply:
Okay.  I had to double check throughout the code but it looks like if it is defined it will always be a `ConstantProperty`.  So we can safely just do `} else if (!billboard.image.getValue()) {`.  I was just being cautious.

Reply:
@hpinkos Ok sounds good.  Thanks for checking it out!